src/crepe/system/InputSystem.cpp

```diff
@@ -0,0 +1,154 @@
+#include "../api/Button.h"
+#include "../api/EventManager.h"
+#include "ComponentManager.h"
+
+#include "system/InputSystem.h"
+
+using namespace crepe;
+
+void InputSystem::update() {
+	EventManager & event_mgr = EventManager::get_instance();
+	std::vector<SDLContext::EventData> event_list = SDLContext::get_instance().get_events();
+
+	for (const SDLContext::EventData & event : event_list) {
+		switch (event.event_type) {
```

Comment:
Ik zie dat je een mooie lijst aan events hebt. Is het een goed idee om dat te documenteren voor de game programmeur?

Reply:
die lijst aan events is eigenlijk meer om de events door te geven vanuit de facade naar het input system de events die de user gebruikts staat in de Events.h